contrib_bots/lib/CommuteBot/CommuteBot.py

```diff
@@ -0,0 +1,215 @@
+        original_content = message['content']
+        content_list = original_content.split()
+
+        if "help" in content_list:
+            help_info = '''
+    Obligatory Inputs:
+        Origin e.g. origins=New+York,NY,USA
+        Destination e.g. destinations=Chicago,IL,USA
+        Mode Of Transport (inputs: driving, walking, bicycling, transit)
+        e.g. mode=driving or mode=transit
+    Optional Inputs:
+        Units (metric or imperial)
+        e.g. units=metric
+        Restrictions (inputs: tolls, highways, ferries, indoor)
+        e.g. avoid=tolls
+        Departure Time (inputs: now or (YYYY, MM, DD, HH, MM, SS) departing)
+        e.g. departure_time=now or departure_time=2016,12,17,23,40,40
+        Arrival Time (inputs: (YYYY, MM, DD, HH, MM, SS) arriving)
+        e.g. arrival_time=2016,12,17,23,40,40
+        Languages:
+        Languages list: https://developers.google.com/maps/faq#languagesupport)
+        e.g. language=fr
+
+    Sample request:
+        @commute origins=Chicago,IL,USA destinations=New+York,NY,USA language=fr
+
+    Please note:
+        Fare information can be derived, though is solely dependent on the
+        availability of the information released by public transport operators.
+        Duration in traffic can only be derived if a departure time is set.
+        If a location has spaces in its name, please use a + symbol in the
+        place of the space/s.
+        A departure time and a arrival time can not be inputted at the same time
+        To add more than 1 input for a category,
+        e.g. more than 1 destinations,
+        use (|), e.g. destinations=Trump+Tower|The+White+House
+        No spaces within addresses.
+        Departure times and arrival times must be in the UTC timezone,
+        you can use the timezone bot.
+                            '''
+            self.send_info(message, help_info, client)
+        else:
+            params = {}
+            # converts input parameters for api into a dictionary
+            # structure for more aethestic coding
+            for item in content_list:
+                # enables key value pair
+                org = item.split('=')
+                # ensures that invalid inputs are not entered into url request
+                if len(org) != 2:
+                    continue
+                key, value = org
+                params.update({key: value})
+
+            # limited to UTC timezone because of difficulty with user inputting
+            # correct string for input
+            if 'departure_time' in params:
+                if params['departure_time'] != 'now':
+                    departure_time = params['departure_time']
+                    times = departure_time.split(',')
+                    times = [int(x) for x in times]
+                    a = dt.datetime(1970, 1, 1, 0, 0, 0)
+                    b = dt.datetime(*times)
+                    total_seconds = str(int((b-a).total_seconds()))
```

Comment:
Yeah, you're totally right, `b.strftime('%s')` would be the next best thing and it's a string.